web-ui/src/app/components/trezor-connect/trezor-connect.component.ts

```diff
@@ -214,4 +363,61 @@ export class TrezorConnectComponent implements OnInit {
   }
 
   refresh() { }
+
+  private base64ToHex(str: string) {
+    const raw = atob(str);
+    let result = '';
+    for (let i = 0; i < raw.length; i++) {
+      const hex = raw.charCodeAt(i).toString(16);
+      result += (hex.length === 2 ? hex : '0' + hex);
+    }
+    return result.toUpperCase();
+  }
+
+  isInt(n: any) {
```

Comment:
This looks hacky, is there a better way?

Reply:
found a better way and fixed